tests/containers/testenv_prepare.pm

```diff
@@ -23,31 +23,44 @@ use containers::container_images;
 
 sub run {
     my ($self) = @_;
-    my $unit_name = 'test_nginx';
 
     select_serial_terminal;
-    my $quadlet = script_output "rpm -ql podman | grep podman/quadlet";
+    my $tumbleweed_container_image = "registry.opensuse.org/opensuse/tumbleweed:latest";
+    my $nginx_container_image = "registry.opensuse.org/opensuse/nginx:latest";
 
     assert_script_run('curl -sLf --create-dirs -vo /home/nginx/nginx.conf ' . data_url('containers/nginx/') . 'nginx.conf');
     assert_script_run('curl -sLf --create-dirs -vo /home/nginx/index.html ' . data_url('containers/nginx/') . 'index.html');
```

Comment:
```suggestion
    assert_script_run('curl -sSLf --create-dirs -vo /home/nginx/index.html ' . data_url('containers/nginx/') . 'index.html');
```
It is ok now, only as simple suggestion: the `-s` option is to remove all ouptput; by adding `-S` we show eventual errors instead, to ease debugging.